### ISSUE TITLE
Fx not showing in Spacarlight

### ISSUE THREAD

**Issue by yuitert** [precision-utwente/spalight#6]
Is it possible to also add Fx to the output list of Spacarlight? We are working with contact models, and there it could be valuable to also know the applied forces on the system. I would even think that we should make a Fexternal  = fxtot + fx. This shows the forces that act on the system, on the boundaries but also prescribed.

**Comment by Marijnn** [comment]
Definitely. I can add that without much delay. What about fxgrav? We might add that to your proposed Fexternal as well then. After all, if someone didn't want the gravity forces, it's easy to turn it off.

**Comment by yuitert** [comment]
Indeed, for most applications it is sufficient to know the net force/torque on a body. However, I expect that there are uses where we want to see one of the individual components. I guess this can be added as an option?

**Comment by Marijnn** [comment]
So, something like:
- `nprops(i).Freac = fxtot`
- `nprops(i).Fext = fxtot+fx+fxgrav`

And then some optional flag in `opt` which, when true, to get the separate components:
- `nprops(i).Fgrav = fxgrav`
- `nprops(i).Fappl = fx`

Would this be good?
